tahrir/templates/admin.mak

```diff
@@ -83,6 +83,100 @@
 
   <div class="clear"></div>
 
+  <!-- TEAM PANEL -->
+
+  <div class="grid-33">
+    <h2>Team</h2>
+    <p>Add a new team</p>
+  </div>
+
+  <div class="grid-66">
+    <form class="admin-form" method="POST">
+      <input type="hidden" name="csrf_token" value="${request.session.get_csrf_token()}"/>
+      <table>
+        <tr>
+          <td><label for="team-name">Team Name</label></td>
+          <td><input name="team-name" required="required" placeholder="Creative Team"/></td>
+        </tr>
+        <tr>
+          <td></td><td>
+            <input name="add-team" type="submit" value="Add Team" />
+          </td>
+        </tr>
+      </table>
+    </form>
+  </div>
+
+  <div class="clear"></div>
+
+  <div class="grid-33">
+    <h2>Series</h2>
+    <p>Add a new Series</p>
+  </div>
+
+  <div class="grid-66">
+    <form class="admin-form" method="POST">
+      <input type="hidden" name="csrf_token" value="${request.session.get_csrf_token()}"/>
+      <table>
+        <tr>
+          <td><label for="series-name">Series Name</label></td>
+          <td><input name="series-name" required="required" placeholder="Awesome Series"/></td>
+        </tr>
+        <tr>
+          <td><label for="series-description">Series Description</label></td>
+          <td><input name="series-description" required="required" placeholder="This is an awesome series"/></td>
+        </tr>
+        <tr>
+          <td><label for="series-tags">Series Tags</label></td>
+          <td><input name="series-tags" required="required" placeholder="awesome, series"/></td>
+        </tr>
+        <tr>
+          <td><label for="series-team-id">Team ID</label></td>
+          <td><input name="series-team-id" required="required" placeholder="Team Series belongs to"/></td>
+        </tr>
+        <tr>
+          <td></td><td>
+            <input name="add-series" type="submit" value="Add Series" />
+          </td>
+        </tr>
+      </table>
+    </form>
+  </div>
+
+  <div class="clear"></div>
+
+  <div class="grid-33">
+    <h2>Perk</h2>
+    <p>Add a new Perk</p>
```

Comment:
Perk isn't quite a word I'm used to or know well, should we add a one-liner description of what perks are?